src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="starRow stars--0">
```

Comment:
classes should be in kebab-case or BEM, not in camelCase

Reply:
BEM modifier should only be used on element which has the corresponding block class